### PR TITLE
feature/17_map_only_job

### PR DESCRIPTION
**Pull request by frbattid**
- Implements issue #17 
- 100% unit tests passed:

```
Results :
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0
```
- (unofficial) e2e tests passed:

```
$ hadoop fs -cat tidoop/input/test_num.txt | more
3.9
12
4
67
8
...
$ hadoop jar target/tidoop-mr-lib-0.0.0-SNAPSHOT-jar-with-dependencies.jar com.telefonica.iot.tidoop.mrlib.MapOnly -libjars target/tidoop-mr-lib-0.0.0-SNAPSHOT-jar-with-dependencies.jar tidoop/input tidoop/output 'double w = x * x; double z = w + 1; String y = z + "_kk"'
15/06/08 18:34:58 INFO input.FileInputFormat: Total input paths to process : 1
15/06/08 18:34:58 WARN snappy.LoadSnappy: Snappy native library is available
15/06/08 18:34:58 INFO util.NativeCodeLoader: Loaded the native-hadoop library
15/06/08 18:34:58 INFO snappy.LoadSnappy: Snappy native library loaded
15/06/08 18:34:59 INFO mapred.JobClient: Running job: job_201409031055_4335
15/06/08 18:35:00 INFO mapred.JobClient:  map 0% reduce 0%
15/06/08 18:35:11 INFO mapred.JobClient:  map 8% reduce 0%
15/06/08 18:35:14 INFO mapred.JobClient:  map 15% reduce 0%
15/06/08 18:35:17 INFO mapred.JobClient:  map 22% reduce 0%
15/06/08 18:35:20 INFO mapred.JobClient:  map 29% reduce 0%
15/06/08 18:35:23 INFO mapred.JobClient:  map 37% reduce 0%
15/06/08 18:35:26 INFO mapred.JobClient:  map 44% reduce 0%
15/06/08 18:35:29 INFO mapred.JobClient:  map 51% reduce 0%
15/06/08 18:35:32 INFO mapred.JobClient:  map 59% reduce 0%
15/06/08 18:35:35 INFO mapred.JobClient:  map 65% reduce 0%
15/06/08 18:35:38 INFO mapred.JobClient:  map 73% reduce 0%
15/06/08 18:35:41 INFO mapred.JobClient:  map 80% reduce 0%
15/06/08 18:35:44 INFO mapred.JobClient:  map 87% reduce 0%
15/06/08 18:35:47 INFO mapred.JobClient:  map 94% reduce 0%
15/06/08 18:35:51 INFO mapred.JobClient:  map 100% reduce 0%
15/06/08 18:35:52 INFO mapred.JobClient: Job complete: job_201409031055_4335
15/06/08 18:35:52 INFO mapred.JobClient: Counters: 17
15/06/08 18:35:52 INFO mapred.JobClient:   Job Counters 
15/06/08 18:35:52 INFO mapred.JobClient:     SLOTS_MILLIS_MAPS=51454
15/06/08 18:35:52 INFO mapred.JobClient:     Total time spent by all reduces waiting after reserving slots (ms)=0
15/06/08 18:35:52 INFO mapred.JobClient:     Total time spent by all maps waiting after reserving slots (ms)=0
15/06/08 18:35:52 INFO mapred.JobClient:     Launched map tasks=1
15/06/08 18:35:52 INFO mapred.JobClient:     Data-local map tasks=1
15/06/08 18:35:52 INFO mapred.JobClient:     SLOTS_MILLIS_REDUCES=0
15/06/08 18:35:52 INFO mapred.JobClient:   FileSystemCounters
15/06/08 18:35:52 INFO mapred.JobClient:     HDFS_BYTES_READ=1968922
15/06/08 18:35:52 INFO mapred.JobClient:     FILE_BYTES_WRITTEN=53271
15/06/08 18:35:52 INFO mapred.JobClient:     HDFS_BYTES_WRITTEN=6246809
15/06/08 18:35:52 INFO mapred.JobClient:   Map-Reduce Framework
15/06/08 18:35:52 INFO mapred.JobClient:     Map input records=598001
15/06/08 18:35:52 INFO mapred.JobClient:     Physical memory (bytes) snapshot=169512960
15/06/08 18:35:52 INFO mapred.JobClient:     Spilled Records=0
15/06/08 18:35:52 INFO mapred.JobClient:     CPU time spent (ms)=52360
15/06/08 18:35:52 INFO mapred.JobClient:     Total committed heap usage (bytes)=177668096
15/06/08 18:35:52 INFO mapred.JobClient:     Virtual memory (bytes) snapshot=734375936
15/06/08 18:35:52 INFO mapred.JobClient:     Map output records=598001
15/06/08 18:35:52 INFO mapred.JobClient:     SPLIT_RAW_BYTES=118
$ hadoop fs -cat tidoop/d/part-m-00000 | more
16.21_kk
145.0_kk
17.0_kk
4490.0_kk
65.0_kk
...
```
- Assignee @fgalan

### CODE REVIEW

File: tidoop-mr-lib/README.md [hunk 1]
```diff
@@ -228,14 +229,76 @@ Usage:
...
-         <output_folder> \
+         <input folder> \
+         <output folder> \
          <regex>
          
 Categories: **transformation**, **generic**, **flat**.
 
 [Top](#top)
 
+###<a name="maponly"></a>MapOnly
```
Comment (fgalan) [inline tidoop-mr-lib/README.md:240]: Maybe is more correct in HTML doing:

```
<a name="maponly">MapOnly</a>
```

...although not sure if that breaks Markdown.

Comment (frbattid) [inline tidoop-mr-lib/README.md:240]: I copied this way of creating an index from PEP proxy... So I do not know if there is any other way to do it :)

Comment (frbattid) [inline tidoop-mr-lib/README.md:240]: NTC

Comment (fgalan) [inline tidoop-mr-lib/README.md:240]: Discussed at skype. NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**